.github/workflows/python-package.yml

```diff
@@ -5,17 +5,19 @@ name: Python package
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, feature/** ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, feature/** ]
 
 jobs:
   build:
 
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        # os: [ubuntu-latest, macos-latest, windows-latest]
+        # disabling windows for now
```

Comment:
The antlr install step would have to be modified to build on windows. Since we're not planning on sticking with the antlr parser, that would be wasted work. Todo: reenable windows build once we remove the antlr dependency